File: android/src/main/kotlin/net/niuxiaoer/flutter_gromore/event/AdEvent.kt
package net.niuxiaoer.flutter_gromore.event

class AdEvent(private val id: String, private val name: String) {

    fun toMap(): HashMap<String, Any> {
        return HashMap<String, Any>().apply {
            set("id", id)
            set("name", name)
        }
    }

}

File: ios/Classes/event/AdEvent.swift
//
//  AdEvent.swift
//  flutter_gromore
//
//  Created by Anand on 2022/6/2.
//

class AdEvent: NSObject {
    private var id: String
    private var name: String
    
    init(id: String, name: String) {
        self.id = id
        self.name = name
    }
    
    func toMap() -> [String: String] {
        return [
            "id": id,
            "name": name
        ]
    }
}
